vector/src/main/java/im/vector/app/features/link/LinkHandlerActivity.kt

```diff
@@ -61,14 +61,18 @@ class LinkHandlerActivity : VectorBaseActivity<ActivityProgressBinding>() {
             return
         }
 
-        if (uri.getQueryParameter(LoginConfig.CONFIG_HS_PARAMETER) != null) {
-            handleConfigUrl(uri)
-        } else if (SUPPORTED_HOSTS.contains(uri.host)) {
-            handleSupportedHostUrl(uri)
-        } else {
-            // Other links are not yet handled, but should not come here (manifest configuration error?)
-            toast(R.string.universal_link_malformed)
-            finish()
+        when {
+            uri.getQueryParameter(LoginConfig.CONFIG_HS_PARAMETER) != null                 -> {
+                handleConfigUrl(uri)
+            }
+            resources.getStringArray(R.array.permalink_supported_hosts).contains(uri.host) -> {
```

Comment:
I didn't understand why you replace `SUPPORTED_HOSTS.` with `resources.getStringArray(R.array.permalink_supported_hosts)`. We may talk about

Reply:
I removed the constant with a string array in the config resource file

Reply:
ok